modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -2852,6 +2852,7 @@ cannot-be-found={0} cannot be found.
 cannot-copy-file-entries-to-the-live-version-of-a-group=Cannot copy file entries to the live version of a group. Make your changes in staging and publish them to Live afterwards to make them public.
 cannot-copy-file-shortcuts-to-the-live-version-of-a-group=Cannot copy file shortcuts to the live version of a group. Make your changes in staging and publish them to Live afterwards to make them public.
 cannot-copy-folders-to-the-live-version-of-a-group=Cannot copy folders to the live version of a group. Make your changes in staging and publish them to Live afterwards to make them public.
+cannot-copy-to-the-live-version-of-a-group=Cannot copy to the live version of a group. Make your changes in staging and publish them to Live afterwards to make them public.
```

Comment:
Maybe it's worth for this new property to replace all similar properties. Or to have input parameter for the type of item to copy.

* cannot-copy-file-entries-to-the-live-version-of-a-group
* cannot-copy-file-shortcuts-to-the-live-version-of-a-group
* cannot-copy-folders-to-the-live-version-of-a-group